Documentation/guides.xml

```diff
@@ -13,7 +13,7 @@
             edit-on-github-branch="main"
             edit-on-github="TYPO3-Documentation/Community-Infrastructure"
             typo3-core-preferred="latest"
-            interlink-shortcode="community-infrastructure"
+            interlink-shortcode="guide-policy"
```

Comment:
I don't understand that. Why? We document the interlink to link to its own repo:
https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/Reference/FileStructure.html

...and that is not guide-policy

Reply:
This is not an extension, but an official manual. Their interlink-shortcodes are registered in render-guides